pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -444,23 +444,6 @@ func updateVersion(ch chan<- prometheus.Metric) {
 	)
 }
 
-type vmiMetrics struct {
```

Comment:
The descriptions are really not needed and should be removed, but I like for instance to store common things like in this case `vmi`, `k8sLabels` and such in a struct. Also not strictly lecessary here, but in general such structs have a few advantages:

 * it shows that some methods/functions belong together. `vmiMetrics.updateMemory`  explains itself more than just `updateMemory`.
 * if needed, mocking is very easy if it is a struct. We have some unit tests which I would prefer not to have since they introduce and overrid global callback functions just to get mock behaviour.
 * It eliminates the need to pass in the same arguments over and over. You could do `vmiMetrics.updateMemory(vmStats)` and have the rest in the struct. Not an issue with not many arguments, but it actually reached the count where I normally start considering adding structs instead of removing them :)

Reply:
Thanks a lot for that review. 

The first thought was that since none of the descriptors were going to be shared somewhere else, there was no need to create a struct for them, I can see that you agree with that.

And I see now that the struct can help to highlight the strong relation between all elements that build these metrics, such as the VMI, k8s labels and the go channel.

If no one shows any downsides to that, I'll proceed to do so